src/components/button/button-component.vue

```diff
@@ -3,7 +3,7 @@
 </style>
 
 <template>
-  <button  class="button" :class="{'buttonBackgroundWhite': whiteBackground, 'buttonBackgroundGreen': greenBackground,}">
+  <button  class="button" :class="{'buttonBackgroundWhite': whiteBackground, 'buttonBackgroundGreen': greenBackground, 'buttonBackgroundBlue': blueBackground,}">
```

Comment:
I'm not sure if this will work, but we may be able to do something cool like this string enumeration, so we could just set the background color based on a string.  Thoughts?

https://github.com/vuejs/vue/issues/7840
```js
    backgroundColor: {
      type: String,
      validator: (val) => ['green', 'white', 'blue'].includes(val)
    },
```